Update model function names (#55)
epiverse-trace/tutorials: episodes/model-choices.Rmd

### Available  models
 
The R package `epidemics` contains functions to run existing models.

> For details on the models that are available, see the package [vignettes](https://epiverse-trace.github.io/epidemics/articles). To learn how to run the models in R, read the documentation using `?epidemics::model_ebola_cpp`. Remember to use the 'Check model equation' questions to help your understanding of an  existing model. 

Reviewer: ```suggestion
For details on the models that are available, see the package [vignettes](https://epiverse-trace.github.io/epidemics/articles). To learn how to run the models in R, read the documentation using `?epidemics::model_ebola_r`. Remember to use the 'Check model equation' questions to help your understanding of an  existing model. 
```

Reviewer: I think is `epidemics::model_ebola_r()` the one that leads to the reference manual. Now `model_ebola_cpp` is still in that reference manual, but as [an internal function not exported to NAMESPACE](https://stackoverflow.com/a/2675532/6702544), and to access it I need to use three double-colon like `epidemics:::model_ebola_cpp()`

> For details on the models that are available, see the package [vignettes](https://epiverse-trace.github.io/epidemics/articles). To learn how to run the models in R, read the documentation using `?epidemics::model_ebola_cpp`. Remember to use the 'Check model equation' questions to help your understanding of an  existing model. 

Reviewer: ```suggestion
For details on the models that are available, see the package [vignettes](https://epiverse-trace.github.io/epidemics/articles). To learn how to run the models in R, read the documentation using `?epidemics::model_ebola_cpp`. Remember to use the questions in the '[Check model equation](#check-model-equations)' checklist to help your understanding of an existing model. 
```

Reviewer: Is this the checklist of questions are you referring to?

Author: Yes, added your suggestion in this [commit](https://github.com/epiverse-trace/tutorials/commit/bcb9a3c5870523e736d642197646237dc6d34e2f) as the suggestion was outdated

Which of the following models would be an appropriate choice for this task:

+  `model_default_cpp()`

> + `model_ebola_cpp()`

Reviewer: ```suggestion
+ `model_ebola_r()`
```

The model is capable of predicting an Ebola type outbreak, but as the model is deterministic, we are not able to explore stochastic variation in the early stages of the outbreak.

> #### `model_ebola_cpp()`

Reviewer: ```suggestion
#### `model_ebola_r()`
```
